site/content/integrate/plugins/webapp/actions.md

```diff
@@ -7,6 +7,194 @@ weight: 11
+
+Retrieves the the specific emoji associated with the supplied `customEmojiName` from the `Redux store`.
+
+
+## Some common client functions
+
+We've listed out some of the commonly-used client functions that you can use in your web app plugin. You can find all the client functions that are available for your plugin to import [in the source code for mattermost-redux](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts).
+
+* ### [getUser = (userId: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L846)
+
+Routes to the user profile of the specified `userId` from the `Mattermost Server`.
+
+
+* ### [getUserByUsername = (username: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L853)
+
+Routes to the user profile of the specified `username` from the `Mattermost Server`.
+
+
+* ### [getChannel = (channelId: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1600)
+
+Routes to the channel of the specified `channelId` from the `Mattermost Server`.
+
+
+* ### [getChannelByName = (teamId: string, channelName: string, includeDeleted = false)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1609)
+
+Routes to the channel of the specified `channelName` from the `Mattermost Server`.
+
+
+* ### [getTeam = (teamId: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1192)
+
+Routes to the team of the specified `teamId` from the `Mattermost Server`.
+
+
+* ### [getTeamByName = (teamName: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1199)
+
+Routes to the team of the specified `teamName` from the `Mattermost Server`.
+
+
+* ### [executeCommand = (command: string, commandArgs: CommandArgs)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L2463)
+
+Executes the specified command with the arguments provided and fetches the response.
+
+
+* ### [getOptions(options: Options) {const newOptions: Options = {...options}](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L440)
+
+Get the client options to make requests to the server. Use this to create your own custom requests.
+
+    ## Custom reducers and actions
+
+Reducers in Redux are pure functions that describe how the data in the store changes after any given action. Reducers will always produce the same resulting state for a given state and action. You can register a custom reducer for your plugin against the Redux store with the `registerReducer` function.
+
+### [registerReducer(reducer)]({{< ref "/integrate/plugins/webapp/reference#registerReducer" >}})
+
+Registers a reducer against the Redux store. It will be accessible in Redux state under `state['plugins-<yourpluginid>']`. It generally accepts a reducer and returns undefined.
+=======
```

Comment:
Should this be removed? Is it serving a function in the docs?

Reply:
That was from the merge conflict resolution - I overlooked it - thanks!